### PR TITLE
fix(console): app insights wrapper direct children style

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

fix app insights wrapper direct children style in AC main pages.

## Testing

Tested locally.
Previous:
image
After fix:
image

## Checklist

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/console/src/containers/ConsoleContent/index.module.scss [hunk 1]
```diff
@@ -16,7 +16,7 @@
   }
 
   // App Insights wrapper on cloud env prevents the CSS assignment to direct children
-  .appInsightsWrapper > * {
+  [class='appInsightsWrapper'] > * {
```
Comment (gao-sun) [inline packages/console/src/containers/ConsoleContent/index.module.scss:19]: just curious didn't this test before?

**Review by simeng-li — APPROVED** [review]
LGTM

File: packages/console/src/containers/ConsoleContent/index.module.scss (hunk 1 above)
Comment (darcyYe) [reply packages/console/src/containers/ConsoleContent/index.module.scss:19]: Manually added a div wrapper in the Chrome console and tested and bypassed the condition of creating `appInsights` instance. Seems this does not reproduce the same environment running on the cloud.

**Review by charIeszhao — APPROVED** [review]

File: packages/console/src/containers/ConsoleContent/index.module.scss (hunk 1 above)
Comment (gao-sun) [reply packages/console/src/containers/ConsoleContent/index.module.scss:19]: cool. remember to repro locally as we have the connection string now

**Pull request merged**